src/components/AdminSettings.vue

```diff
@@ -161,6 +161,143 @@
 				</div>
 			</div>
 		</div>
+		<div class="group-folder-setup">
+			<FormHeading index="4"
+				:is-group-folder-setup-heading="true"
+				:title="t('integration_openproject', 'Managed project folders (recommended)')"
+				:is-setup-complete-without-group-folders="isSetupCompleteWithoutGroupFolders"
+				:is-group-folder-setup-correct-after-setup="isGroupFolderSetupCorrectAfterSetup"
+				:is-complete="isGroupFolderSetupCompleted"
+				:is-disabled="isGroupFolderSetUpInDisableMode" />
+			<div v-if="showDefaultManagedFolders">
+				<div v-if="isGroupFolderSetupFormInEdit">
+					<NcCheckboxRadioSwitch type="switch" :checked.sync="isProjectFolderSwitchEnabled" @update:checked="changeGroupFolderSetUpState">
+						<b>{{ t('integration_openproject', 'Automatically managed folders') }}</b>
+					</NcCheckboxRadioSwitch>
+					<div v-if="isProjectFolderSwitchEnabled === false" class="complete-without-groupfolders">
+						<p class="group-folder-description">
+							{{
+								t('integration_openproject', 'We recommend using this functionality but is not mandatory. Please activate in case you want to use the automatic creation and management of project folders.')
+							}}
+						</p>
+						<div class="form-actions">
+							<NcButton type="primary"
+								data-test-id="complete-without-group-folder-form-btn"
+								@click="completeIntegrationWithoutGroupFolderSetUp">
+								<template #icon>
+									<CheckBoldIcon :size="20" />
+								</template>
+								{{
+									textLabelGroupFolderSetupButton
+								}}
+							</NcButton>
+						</div>
+					</div>
+					<div v-else>
+						<p class="group-folder-description">
+							{{
+								t('integration_openproject', 'Let OpenProject create folders per project automatically. It will ensure that every team member has always the correct access permissions.')
+							}}
+						</p>
+						<br>
+						<b>{{ t('integration_openproject', 'OpenProject user, group and folder') }}</b>
+						<p class="group-folder-description">
+							{{
+								t('integration_openproject', 'For automatically managing project folders, this app needs to setup a special group folder, assigned to a group and managed by a user, each called "OpenProject".')
+							}} <br>
+							{{
+								t('integration_openproject', 'The app will never delete files or folders, even if you deactivate this later')
```

Comment:
We need a full-stop here
```suggestion
								t('integration_openproject', 'The app will never delete files or folders, even if you deactivate this later.')
```